lib/src/executables/react16_upgrade.dart

```diff
@@ -51,7 +51,7 @@ void main(List<String> args) {
     AggregateSuggestor([
       PubspecReactUpdater(reactVersionConstraint, shouldAddDependencies: false),
       PubspecOverReactUpgrader(overReactVersionConstraint,
-          shouldAddDependencies: false)
+          shouldAddDependencies: true),
```

Comment:
Why was this changed?

Reply:
Item 1 in the PR description @greglittlefield-wf

Reply:
:doh: 👍